### ISSUE TITLE
 Tdarr_Plugin_a9hd_FFMPEG_Transcode_Specific_Audio_Stream_Codecs - 7.1 tracks lose a channel (5.0)

### ISSUE THREAD

**Issue by gurpalw** [HaveAGitGat/Tdarr_Plugins#665]
when converting a DTS 7.1 track to EAC3, the output file is only 5.0(not 5.1)

The same thing does not occur to DTS 5.1 tracks when they are converted to EAC3.

settings for plugin are DTS --> EAC3 @640k

**Comment by HaveAGitGat** [comment]
That's odd, can see the command here:
https://github.com/HaveAGitGat/Tdarr_Plugins/blob/27f0f3196b8a7a5d210ae70cb49c023229fc4a69/Community/Tdarr_Plugin_a9hd_FFMPEG_Transcode_Specific_Audio_Stream_Codecs.js#L112

So no `-ac` command to set the audio channels so would assume it uses the same as the input which it usually does.

**Comment by gurpalw** [comment]
> That's odd, can see the command here:
> 
> https://github.com/HaveAGitGat/Tdarr_Plugins/blob/27f0f3196b8a7a5d210ae70cb49c023229fc4a69/Community/Tdarr_Plugin_a9hd_FFMPEG_Transcode_Specific_Audio_Stream_Codecs.js#L112
> 
> So no `-ac` command to set the audio channels so would assume it uses the same as the input which it usually does.

indeed. the same thing happens when converting from DTS 7.1 to FLAC. it always results in 5.0. i had to add -ac 6 to fix it.

**Comment by gurpalw** [comment]
any update on this please? were you able to recreate?
